homeassistant/components/yeelight/light.py

```diff
@@ -562,7 +564,8 @@ def device_state_attributes(self):
         """Return the device specific state attributes."""
 
         attributes = {
-            "flowing": self.device.is_color_flow_enabled,
+            PROPERTY_COLOR_MODE: self.device.color_mode,
+            PROPERTY_FLOWING: self.device.is_color_flow_enabled,
```

Comment:
I don't think use `PROPERTY` constants as attribute names is a good idea. They just happen to be the same.

Reply:
Why that's bad ? Idea was for easier debugging, so same property is reported on HA light card and for example when requested via python-yeelight cli etc. I wanted to avoid any confusion, whether show property is the same as in cli / yeelight api docs etc.

Reply:
I mean, ideally, we should use `ATTR_` constants for attribute names. But since you're intentionally making them the same, I guess this is fine. Just an advice, it's up to you to device.

Reply:
My understanding also was that ATTR_ should be used in yaml / config flow ? Not sure about guidelines here. I hope someone from HA core team will take a look and provide best way to proceed this. As you are codeowner, can you make your own code review ? Maybe this could speed up process, as it will be initially reviewed already ?